app/bundles/ConfigBundle/Form/Type/ConfigType.php

```diff
@@ -23,17 +24,16 @@ public function __construct(
 
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
-        // TODO very dirty quick fix for https://github.com/mautic/mautic/issues/8854
-        if (isset($options['data']['apiconfig']['parameters']['api_oauth2_access_token_lifetime'])
-            && 3600 === $options['data']['apiconfig']['parameters']['api_oauth2_access_token_lifetime']
-        ) {
-            $options['data']['apiconfig']['parameters']['api_oauth2_access_token_lifetime'] = 60;
+        $parameterLoader = new ParameterLoader();
+        $parameters      = $parameterLoader->getLocalParameterBag();
```

Comment:
Is there any reason why we need to read the config files again instead of using the `CoreParametersHelper` service that could be provided via a constructor dependency?

Reply:
Wow, that's so messy. I'm wondering whether we should just create a migration to convert the numbers in local.php to seconds and remove all the recalculations. WDYT?